Makefile

```diff
@@ -0,0 +1,19 @@
+CRYSTAL_BIN ?= $(shell which crystal)
+PREFIX ?= /usr/local
+SHARD_BIN ?= ../../bin
+
+build:
+	$(CRYSTAL_BIN) build --no-debug -o bin/neph src/bin/neph_bin.cr $(CRFLAGS)
+clean:
+	rm -f ./bin/neph
+install: build
+	mkdir -p $(PREFIX)/bin
+	cp ./bin/neph $(PREFIX)/bin
```

Comment:
Maybe the user don't have enough permission, so `sudo` may be needed.
Some users may prefer to install binaries *only* using the distro package manager. I think this job isn't needed. If someone wants it installed system wide, maybe a better way to do it with the distro package manager.
Also, this `Makefile` is designed to be used when `neph` is installed as a project shard dependency. It is unlikely to happen to install Neph system wide from a shard dependency directory of a Crystal project. An easier way to clone the Neph repository directly, and not by `shards`.